exercises/clock/src/test/java/ClockEqualTest.java

```diff
@@ -1,98 +1,98 @@
 import org.junit.Ignore;
 import org.junit.Test;
 
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertEquals;
 
 public class ClockEqualTest {
 
     @Ignore("Remove to run test")
     @Test
     public void clocksWithSameTimeAreEqual() {
-        assertTrue(new Clock(15, 37).equals(new Clock(15, 37)));
+        assertEquals(new Clock(15, 37), (new Clock(15, 37)));
```

Comment:
I agree that this is a good change :) But it might make it harder for people to see what they need to implement to make the test pass. Maybe it's worth adding a `HINT.md` file explaining a bit about overriding the `equals` method?